grisera_api/setup.py

```diff
@@ -0,0 +1,23 @@
+from channel.channel_service import ChannelService
+from channel.channel_model import ChannelIn
+import os
+from time import sleep
+
+
+class SetupNodes:
+    """
+    Class to init nodes in grahp database
+    """
+    def set_channels(self):
+        """
+        Initialize values of channels
+        """
+        channel_service = ChannelService()
+        channel_types = ["Audio", "BVP", "Chest size", "Depth video", "ECG",
+                         "EDA", "EEG", "EMG", "RGB video", "Temperature"]
```

Comment:
I think it'd be better to store list of available channel types in some other structure (class, enum?), not directly  in the method.